components/director/run.sh

```diff
@@ -169,13 +170,12 @@ else
         cat ${ROOT_PATH}/../schema-migrator/seeds/director/*.sql | \
             docker exec -i ${POSTGRES_CONTAINER} psql -U "${DB_USER}" -h "${DB_HOST}" -p "${DB_PORT}" -d "${DB_NAME}"
     else
-        if [[ ! -f ${ROOT_PATH}/../schema-migrator/seeds/dump.sql ]]; then
+        if [[ ! -d ${ROOT_PATH}/../schema-migrator/seeds/dump-local ]]; then
             echo -e "${GREEN}Will pull DB dump from GCR bucket${NC}"
-            gsutil cp gs://sap-cp-cmp-dev-db-dump/dump.sql ${ROOT_PATH}/../schema-migrator/seeds/dump.sql
+            gsutil cp -r gs://sap-cp-cmp-dev-db-dump/dump-local ${ROOT_PATH}/../schema-migrator/seeds
```

Comment:
What is the local dump? Why we don't use the schema version here as well?

Reply:
It looks like this feature https://github.com/kyma-incubator/compass/pull/2456 missed the directors run.sh script, maybe we can unify them as it will not work anymore (it is not working since the PR above)